src/components/Text/Text.tsx

```diff
@@ -22,7 +23,8 @@ export type Props = {
   /**
    * Controls whether to render text inline (defaults to "p");
    */
-  as?: 'p' | 'span';
+  as?: 'p' | 'span' | 'div';
```

Comment:
`as='div'` indicates use of a text passage

Reply:
Maybe add a comment about that, something about using a `div` if you have `p` tags in the text?

Reply:
Sounds good, will do